context/app/static/js/components/home/HuBMAPDatasetsChart/HuBMAPDatasetsChart.jsx

```diff
@@ -1,13 +1,18 @@
-import React, { useState } from 'react';
-import { scaleLinear, scaleOrdinal, scaleBand } from '@visx/scale';
-import { LegendOrdinal } from '@visx/legend';
+import React from 'react';
 
+import ChartWrapper from 'js/shared-styles/charts/ChartWrapper';
 import useSearchData from 'js/hooks/useSearchData';
-import { useChartPalette, useAssayTypeBarChartData } from './hooks';
-import { getAssayTypesCompositeAggsQuery } from './utils';
-import AssayTypeBarChart from '../AssayTypeBarChart/AssayTypeBarChart';
-import { Flex, ChartWrapper, LegendWrapper } from './style';
-import AssayTypeBarChartDropdown from '../AssayTypeBarChartDropdown';
+import { useSelectedDropdownIndex } from 'js/shared-styles/dropdowns/DropdownListbox';
```

Comment:
Not a big deal, but generally the non-default exports from capitalized components are kind of private, I think?

Reply:
The component is nested within the same conditional used for the `Assays` component on the Organ page which I believe has the same conditions you would want for this component.